src/v/kafka/server/server.cc

```diff
@@ -131,7 +132,9 @@ server::server(
   , _gssapi_principal_mapper(
       config::shard_local_cfg().sasl_kerberos_principal_mapping.bind())
   , _krb_configurator(config::shard_local_cfg().sasl_kerberos_config.bind())
-  , _thread_worker(tw) {
+  , _thread_worker(tw)
+  , _replica_selector(
+      std::make_unique<select_closest_follower>(_metadata_cache.local())) {
```

Comment:
this should be exposed in a broker config no? (with default being leader selector).

Reply:
it depends on how we define it. For now we do not support different replica selectors and if a consumer is setting up the `rack.id` field in the `FetchRequest` it express the intent to fetch from the follower. I didn't want it to be necessary to make the changes in two places (broker & client) to enable follower fetching.